mne/viz/_3d.py

```diff
@@ -2613,7 +2613,8 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     if colors is None:
         colors = _get_color_list()
 
-    linestyles = ['-', '--', ':']
+    linestyles = cycle(['-', '--', ':'])
+    linestyles = [next(linestyles) for _ in range(len(stcs))]
```

Comment:
Can you exploit the bigger diversity of matplotlib line styles 

https://matplotlib.org/stable/gallery/lines_bars_and_markers/linestyles.html